webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,8 +164,29 @@
                                 <input type="hidden" class='ajax-signature' name="ajax-signature-1" value="{{ crash.signature }}" />
                             </div>
                             <div class="signature-icons">
+                                {% if crash.startup_count > 0 %}
+                                {% if crash.startup_count == crash.count %}
```

Comment:
Is that realistic? 
Also, if some (e.g. 5%) crashes for this signature are not included, it does show the yellow flag after all. And the yellow flag says "Potential Startup Crash".